app/views/auth/two-factor-auth.blade.php

```diff
@@ -0,0 +1,31 @@
+@extends('layout.clean')
+
+@section('content')
+    <div class="login">
+        <div class="col-xs-12 col-xs-offset-0 col-sm-6 col-sm-offset-3 col-md-4 col-md-offset-4 text-center">
+            <div class="welcome-logo">
+                <img class="logo" height="50" src="{{ url('img/cachet-logo.svg') }}" alt="Cachet"/>
```

Comment:
It's also odd. That helper function works for me on styleci and on bootstrap cms, using L4.2 and L5.0.